src/Components/ImagesTable/ImageLink.js

```diff
@@ -26,20 +26,25 @@ const ProvisioningLink = ({ imageId, isExpired, isInClonesTable }) => {
   );
 
   if (!error) {
-    image = image.isClone ? parent : image;
+    const provisionedImage = image.isClone ? parent : image;
     return (
       <Suspense fallback="loading">
-        <Button variant="link" isInline onClick={() => openWizard(true)}>
+        <Button
+          isDisabled={image.status !== 'success'}
```

Comment:
Can we move this to/solve this in ImageLink directly? It currently returns null if there's no uploadstatus, but might as well solve it for all cases, and require that the uploadstatus is defined and has succeeded.